### ISSUE TITLE
All Songs From Playlist Not Shown

### ISSUE THREAD

**Issue by anxiouslelimo** [maxrave-dev/SimpMusic#374]
**Describe the bug**
For some albums many songs are missing from the playlist

**To Reproduce**
1. Search for Bollywood
2. Go to Featured Playlist
3. Select the playlist Bollywood Hitlist
4. Only 24 songs are shown instead of 50

**Expected behavior**
All 50 songs should be shown

**Comment by CrazyWolf13** [comment]
Experiencing this too, only 50/151 were displayed in my case, with no obvious wy to refresh.

**Comment by maxrave-dev** [comment]
Hmm, reproduce but all 50 songs shown

**Comment by CrazyWolf13** [comment]
@maxrave-dev yes I was able to re-add the playlist by removing it and copying again to local, which worked, though there still has to be some sort of reason why they got out of sync, but it's fine for me if we have a workaround.
